migrations/2018_03_16_000000_make_settings_value_nullable.php

```diff
@@ -26,7 +26,7 @@ public function up()
     public function down()
     {
         Schema::table('settings', function (Blueprint $table) {
-            $table->text('value')->nullable(false)->change();
+            $table->text('value')->nullable()->change();
```

Comment:
This does not do anything because the field is nullable already at this point.
Also, the down method should exactly do what it already does, to revert the field being nullable